packages/hoppscotch-common/src/components/importexport/ImportExportList.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="flex flex-col">
+    <HoppSmartExpand>
+      <template #body>
+        <HoppSmartItem
+          v-for="importer in importers"
+          :key="importer.id"
+          :icon="importer.icon"
+          :label="t(`${importer.name}`)"
+          @click="emit('importer-selected', importer.id)"
+        />
+      </template>
+    </HoppSmartExpand>
+    <hr />
+    <div class="flex flex-col space-y-2">
+      <template v-for="exporter in exporters" :key="exporter.id">
+        <!-- adding the title to a span if the item is visible, otherwise the title won't be shown -->
+        <template v-if="exporter.disabled && exporter.title">
```

Comment:
Do we need the `template` wrappers? Can't the `v-if/else` directives be applied straightaway to the direct children?